### PR TITLE
(Feat): If pipeline not present then display `BAN `-icon in details pannel

### PR DESCRIPTION
**Pull request by bhaskarhc**
**What this PR does / why we need it**:
 - If  Pipelines Not-Avaliable then pipeline show :no_entry_sign: -icon instead of :doughnut:(doughnut) graph.
 - :no_entry_sign: (ban-icon ) OnClick ban icon will display Connection error image in DetailPannel .
 - Height fixed for ci-dashboard & Detail pannel  dashboard .
 - Sidebar mayadata color change to maya-purple color .

### Pipeline not avaliable :no_entry_sign: 
Screenshot from 2019-03-12 11-40-05
### Actual pipeline display by click :doughnut: 
Screenshot from 2019-03-15 11-30-40

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by mayavuser** [comment]
Use this colour (004ED6) for Mayadata in the sidebar.

File: src/app/table/table.component.ts [hunk 1]
```diff
@@ -324,6 +325,8 @@ export class TableComponent implements OnInit {
   }
 
   detailPannel(index, pipelineData, buildData, cloud) {
+    console.log("CLOUD:::--",cloud,"NAME::--",this.name,"STATUS::--",this.status);
```
Comment (chandankumar4) [inline src/app/table/table.component.ts]: Remove this line

**Comment by chandankumar4** [comment]
image


The selected pipeline details is not available but it still display the previous details.

File: src/app/table/table.component.ts [hunk 2]
```diff
@@ -358,7 +366,7 @@ export class TableComponent implements OnInit {
     this.pullRequest = this.pullRequestURL(buildData[index])
     this.commitUser = this.commitUsr(buildData[index].jobs[0].author_name, pipelineData[index].sha);
     this.rating = this.ratingCalculation(pipelineData[index].jobs)
-    this.gitlabPipelineUrl = pipelineData[index].web_url;
+    this.gitlabPipelineUrl = this.gitlabpipeURL(pipelineData[index].web_url);
```
Comment (chandankumar4) [inline src/app/table/table.component.ts]: function name would be like :- gitlabPipelineURL

File: src/app/table/table.component.html [hunk 3]
```diff
@@ -138,8 +141,25 @@ <h3 *ngIf="undefined !== buildData" style="color:white">Builds in last one week:
...
-    <div class="detail container border">
+  <div class="col-sm-4 px-0">
+
+    <!-- Pipeline details not present START -->
+    <div class="detail container border " *ngIf="name == 'NA'" >
+        <app-lodding-spinners  *ngIf="showSpinnerDetails; else hideDetailPanelNA"></app-lodding-spinners>
+        <ng-template #hideDetailPanelNA>
+      <div class="pt-5 text-center detailNA" style="vertical-align: middle;">
+        <div class="">
+          <img src="/assets/images/icons/connerr.svg" alt="" height="120">
```
Comment (chandankumar4) [inline src/app/table/table.component.html]: Gave the appropriate file name : connection_error

File: src/app/table/table.component.html [hunk 4]
```diff
@@ -138,8 +137,25 @@ <h3 *ngIf="undefined !== buildData" style="color:white">Builds in last one week:
 
 
   <!-- Detail pannel start -->
-  <div class="col-sm-4 px-0">   
-    <div class="detail container border">
+  <div class="col-sm-4 px-0">
+
+    <!-- Pipeline details not present START -->
+    <div class="detail container border " *ngIf="name == 'NA'" >
```
Comment (inyee786) [inline src/app/table/table.component.html]: you can use if and else here

**Pull request merged**